README.md

```diff
@@ -1,13 +1,19 @@
 # Project Movies
+sing APIs in React - combining useState with useEffect
+ The task was to creat an app that should have at least two pages - one showing a list of movies and one showing details and follow the design from an example.
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Used:
+POSTMAN
+React.js
+Css
+API (themoviedb.org.)
+Devtools
```

Comment:
Really nice addition in the readme, to list all the technologies used like this ⭐️